src/integrations/xiaomi-mi/xiaomi-mi.module.ts

```diff
@@ -0,0 +1,16 @@
+import { DynamicModule, Module } from '@nestjs/common';
+import { XiaomiMiService } from './xiaomi-mi.service';
+import { EntitiesModule } from '../../entities/entities.module';
+import { ConfigModule } from '../../config/config.module';
+import { ScheduleModule } from '@nestjs/schedule';
+
+@Module({})
+export default class XiaomiMiModule {
+  static forRoot(): DynamicModule {
+    return {
+      module: XiaomiMiModule,
+      imports: [EntitiesModule, ConfigModule, ScheduleModule.forRoot()],
```

Comment:
I don't see the `ScheduleModule` being used anywhere by this module, so I would remove it from the imports

Reply:
Removed